### PR TITLE
Awais786/ecom 5109 copy clipboard

### PR DESCRIPTION
**Pull request by awais786**
Adding clipboard.js for copy to clipboard functionality.
Cleanup in existing HTML.
Added new template for copy link. It will be updated in future according to the final css.

### CODE REVIEW

**Comment by awais786** [comment]
@waheedahmed Please review this PR.

**Comment by waheedahmed** [comment]
Can we check if field has value then we add copy button?

**Comment by waheedahmed** [comment]
One minor comment otherwise LGTM 👍.

**Comment by awais786** [comment]
@bderusha please review this PR.

File: course_discovery/static/sass/publisher/course_detail.scss [hunk 1]
```diff
@@ -89,6 +95,21 @@ nav {
     .heading {
       font-weight: bold;
       font-size: 16px;
+
+      .btn-copy {
+        text-decoration: none;
+        margin-left: 10px;
```
Comment (bderusha) [inline course_discovery/static/sass/publisher/course_detail.scss]: `RTL` support

File: course_discovery/templates/publisher/course_run_detail/clipboard.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<a href="#" class="btn-copy"></a>
```
Comment (bderusha) [inline course_discovery/templates/publisher/course_run_detail/clipboard.html]: Don't use anchor tags for non-links, please.  Bad accessibility pattern.  Your class even calls it a button, use the button tag please.

**Comment by bderusha** [comment]
One comment. 👍 once addressed

**Comment by bderusha** [comment]
For something like this, a screenshot of what it looks like would be helpful

**Comment by awais786** [comment]
@bderusha screenshot is attached. For the time being I m using plus sign. But once icons issue is resolved i will replace this with copy icon.

screen shot 2016-08-09 at 4 34 58 pm

File: course_discovery/templates/publisher/course_run_detail/_all.html [hunk 3]
```diff
@@ -2,163 +2,227 @@
 
 <div class="course-information">
     <div class="info-item">
-        <div class="heading">{% trans "Title" %}</div>
-        <div>{{ object.title }}</div>
+        <div class="heading">{% trans "Title" %}
```
Comment (bderusha) [inline course_discovery/templates/publisher/course_run_detail/_all.html]: NIT: `{% trans "Title" %}` should be on next line, properly indented.  Here and all instances below

**Comment by bderusha** [comment]
👍 with quick NIT update

**Comment by bderusha** [comment]
@awais786 Can you create a ticket to add clipboard icon later so we don't lose it?

**Comment by awais786** [comment]
@bderusha I have added the following ticket.
https://openedx.atlassian.net/browse/ECOM-5205

**Pull request merged**